test/unit/spec/appWidgets/input/selectInputSpec.js

```diff
@@ -1,179 +1,235 @@
+                original: {
+                    dropdown_options: {
+                        multiselection: config.multiselection,
+                        options: dropdownOptions,
+                    },
+                },
             },
-            channelName: _bus.channelName,
-            initialValue: 'apple',
+            channelName: config.bus.channelName,
+            initialValue: config.initialValue,
         };
     }
 
     describe('The Select input', () => {
-        beforeEach(() => {
+        beforeEach(function () {
             runtime = Runtime.make();
-            container = document.createElement('div');
             bus = runtime.bus().makeChannelBus({
                 description: 'select input testing',
             });
-            testConfig = buildTestConfig(required, defaultValue, bus);
+            this.container = document.createElement('div');
+            this.testConfig = buildTestConfig({ bus });
         });
 
         afterEach(() => {
             bus.stop();
             runtime.destroy();
-            container.remove();
             TestUtil.clearRuntime();
         });
 
         it('should be defined', () => {
             expect(SelectInput).not.toBeNull();
         });
 
-        it('should be instantiable', () => {
-            const widget = SelectInput.make(testConfig);
+        it('should be instantiable', function () {
+            const widget = SelectInput.make(this.testConfig);
             expect(widget).toEqual(jasmine.any(Object));
             ['start', 'stop'].forEach((fn) => {
                 expect(widget[fn]).toEqual(jasmine.any(Function));
             });
         });
 
-        it('Should start and stop a widget', () => {
-            const widget = SelectInput.make(testConfig);
+        it('Should start and stop a widget', function () {
+            const widget = SelectInput.make(this.testConfig);
 
             return widget
-                .start({ node: container })
+                .start({ node: this.container })
                 .then(() => {
                     // verify it's there.
-                    const inputElem = container.querySelector('select[data-element="input"]');
+                    const inputElem = this.container.querySelector('select[data-element="input"]');
                     expect(inputElem).toBeDefined();
                     return widget.stop();
                 })
                 .then(() => {
                     // verify it's gone.
-                    expect(container.childElementCount).toBe(0);
+                    expect(this.container.childElementCount).toBe(0);
                 });
         });
 
-        it('Should update value via bus', () => {
-            // select one value then update it.
-            const widget = SelectInput.make(testConfig);
-            return widget.start({ node: container }).then(() => {
+        describe('bus updates', () => {
+            it('Should update value via bus', async function () {
```

Comment:
this and the following test were not testing anything as-was, so I added a dev mode and a `set-value` message so that we can test that the widget has received the update and acted accordingly.